controllers/openshift_controller.go

```diff
@@ -1572,7 +1617,10 @@ func (r *KataConfigOpenShiftReconciler) SetupWithManager(mgr ctrl.Manager) error
 		Watches(
 			&corev1.Node{},
 			&NodeEventHandler{r}).
-		Complete(r)
+		Watches(
+			&corev1.ConfigMap{},
+			&ConfigMapEventHandler{r},
```

Comment:
It needs to filter out events and only act on specific configmap. Otherwise it'll be too noisy.
An example is in the credentials_controller.go

Reply:
Yes, I'm going to submit a Predicate version soon.